src/test/java/bio/terra/common/SynapseUtils.java

```diff
@@ -243,14 +243,14 @@ public void synapseTestCleanup() {
     } catch (Exception ex) {
       logger.warn("[Cleanup exception] Unable to drop scoped credentials", ex.getMessage());
     }
-    for (String storageAccountId : storageAccountIds) {
-      try {
-        client.storageAccounts().deleteById(storageAccountId);
-      } catch (Exception ex) {
-        logger.warn("[Cleanup exception] Unable to delete storage account", ex.getMessage());
-      }
+    try {
+      storageAccountIds.forEach(
```

Comment:
I wonder if we should move the try catch inside of the loop that way even if on SA fails to delete, others will get cleared out

Reply:
Great point - thanks for catching that.